internal/controllers/core/kubernetesdiscovery/portforwards.go

```diff
@@ -116,9 +121,12 @@ func (r *Reconciler) toDesiredPortForward(kd *v1alpha1.KubernetesDiscovery) (*v1
 //
 // TODO(nick): This is old legacy behavior, and I'm not totally sure it even
 // makes sense. I wonder if we should just insist that ContainerPort is populated.
-func populateContainerPorts(pf *v1alpha1.PortForward, pod *v1alpha1.Pod) {
+func populateContainerPorts(pft *v1alpha1.PortForwardTemplateSpec, pod *v1alpha1.Pod) []v1alpha1.Forward {
```

Comment:
do you expect that we will still want to print this warning after we change the default? is there any reason not to change the default now?

Reply:
Good question, I was taking a fairly cautious approach here and adding the warning for now before changing the default. At that point, I could see an argument for keeping it to let users know that Tilt is now picking a different port than it would have in the past in this case, but that also might be overkill, e.g. when we **remove** deprecated functionality, we don't _generally_ leave in a warning if someone tries to use it, we just error out.